apiExamples/customizing.html

```diff
@@ -13,6 +13,9 @@
     background: linear-gradient(180deg, #cdcdcd 50%, slategrey 100%);
     width: 50%;
   }
+  #customizedDrawer::part(backdrop) {
+    background: rgba(255, 255, 0, .2)
```

Comment:
We should never demo using colors that are not tokens as it might create the idea that an engineer can hard code a color which is a big no-no for theme support.